sms/v1/openapi.yaml

```diff
@@ -436,20 +435,30 @@ paths:
           content:
             application/json:
               schema:
-                type: array
-                items:
-                  $ref: "#/components/schemas/PhoneProvisioningArrayResponse"
```

Comment:
Could not test it. Is this correct?

Reply:
I think this instead of `array` should be just
```
 schema:
                $ref: "#/components/schemas/PhoneProvisioningArrayResponse"
```

Reply:
nevermind, you already did it :)